libs/wire-api-federation/client.py

```diff
@@ -0,0 +1,26 @@
+import grpc
+
+from router_pb2 import *
+import router_pb2_grpc
+
+# make sure to run ./services/start-services-only.sh first to have federator and brig up!
+print("starting client...")
+
+channel = grpc.insecure_channel('localhost:8098') # public-facing federator port
+stub = router_pb2_grpc.RouteToInternalStub(channel)
+
+def handle_search(handle):
+    param = QueryParam(key="handle".encode("utf-8"), value=handle.encode("utf-8"))
+    return LocalCall(
+            path="users/by-handle".encode("utf-8"),
+            query=[param],
+            method=GET,
+            component=Brig)
```

Comment:
figuring out how to write this requires knowing the code in Brig's federation api module. That makes it a bit harder. Perhaps we should consider replacing brig's federation rest API with another protobuf file at some point. Not now, but perhaps after we're done with the user handle lookup story and have a bit more insights on how grpc performs.